<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BifurcationProfiles</class>
 <widget class="qMRMLWidget" name="BifurcationProfiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>404</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="centerLineLabel">
       <property name="text">
        <string>Centerline model:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="segmentationLabel">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Segmentation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="centerlineSelector">
       <property name="toolTip">
        <string>Pick the input centerline model.</string>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLModelNode</string>
        </stringlist>
       </property>
       <property name="showChildNodeTypes">
        <bool>false</bool>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>false</bool>
       </property>
       <property name="SlicerParameterName" stdset="0">
        <string>inputCenterline</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="segmentationSelector">
       <property name="toolTip">
        <string>Input segmentation.
The input centerline is expected to be inside the lumen surface.</string>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLSegmentationNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="qMRMLSegmentSelectorWidget" name="segmentSelector">
       <property name="toolTip">
        <string>Select an input segment in the selected segmentation.</string>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="segmentationNodeSelectorVisible">
        <bool>false</bool>
       </property>
       <property name="selectNodeUponCreation">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>segmentationSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>segmentSelector</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BifurcationProfiles</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>centerlineSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BifurcationProfiles</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BifurcationProfiles</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
